scheduling/taskmetadata_manager_test.go

```diff
@@ -0,0 +1,895 @@
+package scheduling
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	sp "github.com/featureform/scheduling/storage_providers"
+)
+
+func TestTaskMetadataManager(t *testing.T) {
+	testFns := map[string]func(*testing.T, TaskMetadataManager){
+		"CreateTask": testCreateTask,
+		// "GetTaskMetadata":  TaskMetadataGet,
```

Comment:
Should be enabled?

Reply:
I have the other tests directly for Memory, will change them when I add ETCD.